test/chat_api_web/controllers/customer_controller_test.exs

```diff
@@ -41,6 +41,15 @@ defmodule ChatApiWeb.CustomerControllerTest do
 
       assert ids == [customer.id]
     end
+
+    test "lists all customers in csv format", %{authed_conn: authed_conn, customer: customer} do
+      resp = get(authed_conn, Routes.customer_path(authed_conn, :index) <> "?format=csv")
+      csv = response(resp, 200)
+
+      assert is_binary(csv)
+      assert csv =~ "\"#{customer.id}\","
```

Comment:
nit - let's expand this a bit to verify that other fields are being properly included 👍